app-libs/form-component/src/layout-components/README.md

```
@@ -0,0 +1,71 @@
+# Layout components
+
+Layout components are the form-building blocks an app is composed of. Each one maps to a component
+type that can be configured in Altinn Studio and rendered by the runtime. Unlike the dumb components
+in `app-components`, layout components are allowed to use utilities like `useTranslation`.
+
+### `common/`
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)